packages/swing-store/src/importer.js

```diff
@@ -81,11 +84,10 @@ export async function importSwingStore(exporter, dirPath = null, options = {}) {
 
   // All the metadata is now installed, and we're prepared for
   // artifacts. We walk `getArtifactNames()` and offer each one to the
-  // submodule, which ignores historical ones (unless
-  // 'includeHistorical' is true), and validates+accepts the
-  // rest. This is an initial import, so we don't need to check if we
-  // already have the data, but the submodule function is free to do
-  // that check if they want.
+  // submodule, which may ignore them according to `artifactMode`, and
+  // validates+accepts the rest. This is an initial import, so we
+  // don't need to check if we already have the data, but the
+  // submodule function is free to do that check if they want.
```

Comment:
```suggestion
  // submodule, which may ignore it according to `artifactMode`, but
  // otherwise validates and accepts it. This is an initial import, so we
  // don't need to check if we already have the data, but the
  // submodule function is free to do such checks.
```